### PR TITLE
[WIP] Screen Design Draft

### PR DESCRIPTION
**Pull request by yuesha-yc**
See docs/screendesigndraft.md

I want to discuss the design before implementing the code.

### CODE REVIEW

**Comment by yuesha-yc** [comment]
Hi, @ustc-zzzz ! I would like to invite you to make any opinions to this draft. I contacted @3TUSK on QQ already.

File: docs/screendesigndraft.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# SignMeUp GUI设计方案
```
Comment (3TUSK) [inline docs/screendesigndraft.md:1]: This file should be part of the `design.md`.

**Review by 3TUSK — CHANGES_REQUESTED** [review]
GUI looks neat!

File: docs/screendesigndraft.md (hunk 1 above)
Comment (yuesha-yc) [reply docs/screendesigndraft.md:1]: I will integrate this later, thanks.

**Comment by yuesha-yc** [comment]
I will start working on this tomorrow. If you still have any concern please let me know.

**Comment by yuesha-yc** [comment]
Sorry I've been a little occupied earlier this week. I will finish this today (I hope)

**Pull request merged**